.travis.yml

```diff
@@ -27,11 +27,11 @@ matrix:
     - os: windows
       language: shell
       before_install:
-       - choco install python3 --version 3.6.8 --no-progress -y
+       - choco install python3 --version 3.7.5 --no-progress -y
```

Comment:
No reason, it was part of the various tests.

Reply:
Hmmm, I don't mind this single line, especially if it's related to the PR. Why not be a bit more modern :) But it was mainly that if there's a reason we need 3.7, it would be good to document.